lib/generators/sincerely/ses_webhook_controller_generator.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails/generators'
+require 'rails/generators/active_record'
+
+module Sincerely
+  module Generators
+    class SesWebhookControllerGenerator < ::Rails::Generators::NamedBase
```

Comment:
I've been using `AWS SES`. We should stick to `AWS SES` or `SES` for the whole codebase.

Reply:
I'm ok with AWS SES